gobblin-modules/gobblin-kafka-09/src/main/java/org/apache/gobblin/kafka/writer/Kafka09DataWriter.java

```diff
@@ -126,4 +138,33 @@ public void flush()
       throws IOException {
     this.producer.flush();
   }
+  
```

Comment:
Please change the formatting (spaces) to be consistent with rest of the codebase